src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements Drawing {
```

Comment:
```suggestion
public class Circle extends Figure {
```
якщо Figure implements Drawing тоді всі його наслідникики автоматично теже будуть імплементити Drawing